odinmain/src/main/kotlin/me/odinmain/features/impl/dungeon/MapInfo.kt

```diff
@@ -0,0 +1,97 @@
+package me.odinmain.features.impl.dungeon
+
+import me.odinmain.features.Category
+import me.odinmain.features.Module
+import me.odinmain.features.settings.Setting.Companion.withDependency
+import me.odinmain.features.settings.impl.*
+import me.odinmain.ui.clickgui.util.ColorUtil.withAlpha
+import me.odinmain.ui.hud.HudElement
+import me.odinmain.utils.render.Color
+import me.odinmain.utils.render.getMCTextWidth
+import me.odinmain.utils.render.mcText
+import me.odinmain.utils.render.roundedRectangle
+import me.odinmain.utils.skyblock.dungeon.DungeonUtils
+
+object MapInfo : Module(
+    name = "Map Info",
+    category = Category.DUNGEON,
+    description = "Displays various information about the current dungeon map"
+) {
+    private val disableInBoss: Boolean by BooleanSetting("Disable in boss", default = true, description = "Disables the information display when you're in boss.")
+    private val remaining: Boolean by DualSetting("Min Secrets", "Minimum", "Remaining", default = false, description = "Display minimum secrets or secrets until s+.")
+    private val unknown: Boolean by DualSetting("Deaths", "Deaths", "Unknown", default = false, description = "Display deaths or unknown secrets. (Unknown secrets are secrets in rooms that haven't been discovered yet. May not be helpful in full party runs.)")
+    val togglePaul: Int by SelectorSetting("Paul Settings", "Automatic", options = arrayListOf("Automatic", "Force Disable", "Force Enable"))
+    private val background: Boolean by BooleanSetting("Background", default = false, description = "Render a background behind the score info")
+    private val color: Color by ColorSetting("Background Color", default = Color.DARK_GRAY.withAlpha(0.5f), true, description = "The color of the background").withDependency { background }
+
+    val hud: HudElement by HudSetting("Hud", 10f, 10f, 1f, false) {
+        if (it) {
```

Comment:
You dont need the if (it) statement if you initialize the variables with placeholder text anyway, then it can just show the last values in examplehud

Also the executor is unnecessary and you shouldnt really need them to be variables either, just use the mcTextAndWidth function and give the values in format strings